app/jinja_filters.py

```diff
@@ -1,13 +1,12 @@
 # coding: utf-8
```

Comment:
`min_value` and `max_value` don't seem to be used anywhere now, except for test_dates.json that still has old piping in it.